syft/sbom/format.go

```diff
@@ -18,20 +19,41 @@ func (f FormatID) String() string {
 	return string(f)
 }
 
+const AnyVersion = ""
+
 type Format interface {
 	ID() FormatID
+	IDs() []FormatID
+	Version() string
 	Encode(io.Writer, SBOM) error
 	Decode(io.Reader) (*SBOM, error)
 	Validate(io.Reader) error
+	String() string
```

Comment:
nit: `fmt.Stringer` could be embedded here instead

Reply:
Done 👍